src/cascade/executor/epiviz_runner.py

```diff
@@ -388,12 +415,19 @@ def main(args):
      5. Put that data into a file and run Dismod-AT on that file.
     """
     ec = make_execution_context()
+
     settings = load_settings(ec, args.meid, args.mvid, args.settings_file)
 
     if settings.model.drill != "drill":
         raise NotImplementedError("Only 'drill' mode is currently supported")
 
     add_settings_to_execution_context(ec, settings)
+    if args.skip_cache:
```

Comment:
This is an interesting option.

Reply:
It's actually important because if you do a lot of testing using manipulated json files it's easy to get the tier3 caches into a state that is inconsistent with the models you are trying to run. And then failure.

Reply:
Makes sense.